ui/components/src/model/graph.ts

```diff
@@ -14,7 +14,7 @@
 import { MouseEventHandler } from 'react';
 import { LineSeriesOption } from 'echarts/charts';
 
-export const PROGRESSIVE_MODE_SERIES_LIMIT = 500;
+export const OPTIMIZED_MODE_SERIES_LIMIT = 500;
```

Comment:
Do we have a note elsewhere in the code explaining what this mode is, why it's needed, and how we came up with 500? If not, it might be worth adding a very short summary in a comment here for future us.